pkg/agent/datamodel/types.go

```diff
@@ -1118,6 +1156,33 @@ func (w *WindowsProfile) GetWindowsDockerVersion() string {
 	return KubernetesWindowsDockerVersion
 }
 
+// GetWindowsDefaultRuntimeHandler get the default containerd runtime handler or return default value
```

Comment:
the same question , why is containerd here?

Reply:
Plan to support two runtime handler hyper-v isolation and process isolation in containerd so you will see that the runtime handler is always related to containerd in current code copied from aks-engine.